docs/DeveloperGuide.adoc

```diff
@@ -295,17 +295,24 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
+|`* * *` |data-oriented person |see an overview of my transactions in an intuitive graph |gain insights at a glance
+
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |student |input my spending into different categories |manage my expenses better
+
+|`* * *` |visual person | see how much budget I have left in each category |cut down on spending as necessary
+
+|`* * *` |student with many friends |split the bill with my friends |know how much I should pay for a meal
+
+|`* * *` |student who forgets to pay his debt on time |be reminded to pay my debt before the deadline |stop incurring interests
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* *` |patriotic Singaporean who travels to JB often |easily calculate how much Ringgit to bring |enjoy my holiday with insufficient money
```

Comment:
sufficient money instead of insufficient money? perhaps a typo?

Reply:
Overall, extremely well done for the User Stories. User Stories are very diversified, covering a good range of Users.